packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1119,7 +1121,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     const { onRenderOption = this._onRenderOptionContent } = this.props;
 
     return (
-      <div key={item.key} className={this._classNames.header} role="heading">
+      <div key={item.key} className={this._classNames.header} aria-level={this.props.label ? 3 : 2} role="heading">
```

Comment:
>this.props.label ? 3 : 2 [](start = 74, length = 24)

Please do leave a comment as to why label informs 3/2 here. To someone without the context that you have, this can look very arbitrary.